terraform/data/test-azure-kubernetes.tfvars

```diff
@@ -1,5 +1,113 @@
 resource_groups = {
-  "test-az-tf-gh" = {
-    location = "westus2"
+    "terraform-bootcamp-rg" = "USGovTexas",
+}
+
+vnets = {
+    name                = "terraform-bootcamp-vnet"
+    address_space       = ["10.0.0.0/23"]
+    location            = "USGovTexas"
+    resource_group_name = "terraform-bootcamp-rg"
+}
+
+subnets = {
+    "terraform-bootcamp-vm-snet" = {
+        resource_group_name  = "terraform-bootcamp-rg"
+        virtual_network_name = "terraform-bootcamp-vnet"
+        address_prefixes     = ["10.0.0.0/27"]
+    },
+    "terraform-bootcamp-pe-snet" = {
+        resource_group_name  = "terraform-bootcamp-rg"
+        virtual_network_name = "terraform-bootcamp-vnet"
+        address_prefixes     = ["10.0.0.32/27"]
+        private_endpoint_network_policies_enabled = false
+    },
+    "AzureBastionSubnet" = {
+        resource_group_name  = "terraform-bootcamp-rg"
+        virtual_network_name = "terraform-bootcamp-vnet"
+        address_prefixes     = ["10.0.0.64/26"]
+    }
+}
+
+pip = {
+    name                = "terraform-bootcamp-pip"
+    location            = "USGovTexas"
+    resource_group_name = "terraform-bootcamp-rg"
+    allocation_method   = "Static"
+    sku                 = "Standard"
+}
+
+bastion = {
+    name                = "terraform-bootcamp-bastion"
+    location            = "USGovTexas"
+    resource_group_name = "terraform-bootcamp-rg"
+    ip_configuration = {
+        name        = "terraform-bootcamp-bast-config"
+        subnet_name = "AzureBastionSubnet"
+  }
+}
+
+
+nic = {
+    name                = "terraform-bootcamp-nic"
+    location            = "USGovTexas"
+    resource_group_name = "terraform-bootcamp-rg"
+    ip_configuration = {
+        name                          = "terraform-bootcamp-nic"
+        subnet_name                   = "terraform-bootcamp-vm-snet"
+        private_ip_address_allocation = "Dynamic"
+  }
+}
+
+virtual_machine = {
+  name                = "terraform-vm"
+  resource_group_name = "terraform-bootcamp-rg"
+  location            = "USGovTexas"
+  size                = "Standard_D4s_v3"
+  admin_username      = "admin01"
+  admin_password      = "!A@S3d4f5g6h7j8k"
```

Comment:
let's use the TF random_password resource for the VM PW. Set the lifecycle ignore block on the VM resource for the admin_password attr as well.